### ISSUE TITLE
conv2d calculation results are inconsistent with pytorch

### ISSUE THREAD

**Issue by lvjunsetup** [tensorflow/tensorflow#39115]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information** 
- Have I written custom code (as opposed to using a stock
example script provided in TensorFlow): 
- OS Platform and Distribution (e.g.,
Linux Ubuntu 16.04): 
- TensorFlow installed from (source or
binary): - TensorFlow version (use command below): 
- Python version: 3.6.5
- CUDA/cuDNN version: - GPU model and memory:10.1

**Describe the current behavior**
For some specific data, the calculation results of tensorflow and pytorch are inconsistent
**Describe the expected behavior**
The calculation results of tensorflow and pytorch are consistent or the difference is small
**Standalone code to reproduce the issue** 

        if target_interface == 'conv1':
            weights_torch = torch.from_numpy(np.full((11, 11, 3, 1), 1, np.float64).transpose((3, 2, 0, 1)))
            output_pytorch_cpu = torch.from_numpy(
                F.conv2d(torch.from_numpy(input_pytorch.numpy().transpose((0, 3, 1, 2))), weights_torch, padding=0,
                         stride=4).numpy().transpose((0, 2, 3, 1)))
**Other info / logs** 
tensorflow_output:
31.42857361,31.4285183,31.42857361
69.24489594,69.24489594,59.3673439
100.89792633,100.89792633,100.89792633
133.26530457,133.26530457,133.26530457
60.22449875,60.22449875,60.22449875
187.46939087,187.46939087,187.46939087
205.918396,205.918396,205.918396
96.42860413,96.42860413,96.42860413
61.42854309,61.42854309,61.42854309
148.89811707,148.89811707,148.89811707
96.28572083,96.28572083,96.28572083
83.02045441,83.02045441,83.02045441
87.89794922,87.89794922,87.89794922
135.12240601,135.12240601,135.12240601

pytorch_output
31.428574,31.428518,31.428574
59.367344,69.244896,69.244896
100.89793,100.89793,100.89793
133.2653,133.2653,133.2653
60.2245,60.2245,60.2245
187.46939,187.46939,187.46939
205.9184,205.9184,205.9184
96.428604,96.428604,96.428604
61.428543,61.428543,61.428543
148.89812,148.89812,148.89812
96.28572,96.28572,96.28572
83.020454,83.020454,83.020454
87.89795,87.89795,87.89795
135.1224,135.1224,135.1224

 The version of tensorflow is 1.90. The error operators are conv2d () and avg_pooling (). Thank you.

This error is a fact, and it can be reproduced by using a large number of random tensors as input to conv2d () or avg_pooling (). Thank you.

**Comment by Saduf2019** [comment]
@lvjunsetup 
We see that the code provided is incomplete, please share simple stand alone code to replicate the issue faced along with the tensorflow version on which the error was faced.

**Comment by lvjunsetup** [comment]
Hello, our project is so big that it is difficult to take out the code alone now, I will sort it out and send it to you.

**Comment by lvjunsetup** [comment]
> @lvjunsetup
> We see that the code provided is incomplete, please share simple stand alone code to replicate the issue faced along with the tensorflow version on which the error was faced.

Hello, our project is so big that it is difficult to take out the code alone now, I will sort it out and send it to you.

**Comment by Saduf2019** [comment]
@lvjunsetup
Also i see you are using an old version of tensorflow , could you please try on later version as 1.15 or 2.1 and let us know if that helps.

**Comment by Saduf2019** [comment]
@lvjunsetup
Please update as per above comment
